src/index.ts

```diff
@@ -45,7 +45,7 @@ export const HTML_LANGUAGES: string[] = [
   'haml',
   'handlebars',
   'html',
-  'HTML (Eex)',
+  'HTML (EEx)',
```

Comment:
We need `HTML (EEx)` _and_ `HTML (Eex)` to handle both the [`mjmcloug.vscode-elixir`](https://marketplace.visualstudio.com/items?itemName=mjmcloug.vscode-elixir) and [`jakebecker.elixir-ls`](https://marketplace.visualstudio.com/items?itemName=JakeBecker.elixir-ls) extensions:

```suggestion
  'HTML (Eex)',
  'HTML (EEx)',
```

Reply:
Thanks!